tests/scenes/atmosphere/test_particle_layer.py

```diff
@@ -4,18 +4,116 @@
 
 from eradiate import path_resolver
 from eradiate import unit_registry as ureg
-from eradiate.contexts import KernelDictContext, SpectralContext
+from eradiate.contexts import KernelDictContext, SpectralContext, MonoSpectralContext
 from eradiate.scenes.atmosphere._particle_dist import UniformParticleDistribution
 from eradiate.scenes.atmosphere._particle_layer import ParticleLayer
 from eradiate.units import to_quantity
 
 
+def to_dataset(albedo, sigma_t, phase, mu, w):
+    return xr.Dataset(
+        data_vars={
+            "sigma_t": (
+                "w",
+                sigma_t.magnitude,
+                dict(
+                    standard_name="air_volume_extinction_coefficient",
+                    units=f"{sigma_t.units:~}",
+                ),
+            ),
+            "albedo": (
+                "w",
+                albedo.magnitude,
+                dict(
+                    standard_name="single_scattering_albedo", units=f"{albedo.units:~}"
+                ),
+            ),
+            "phase": (
+                ("w", "mu", "i", "j"),
+                phase.magnitude,
+                dict(standard_name="scattering_phase_matrix", units=f"{phase.units:~}"),
+            ),
+        },
+        coords={
+            "w": ("w", w.magnitude, dict(units=f"{w.units:~}")),
+            "mu": (
+                "mu",
+                mu.magnitude,
+                dict(standard_name="scattering_angle_cosine", units=f"{mu.units:~}"),
+            ),
+            "i": ("i", [0]),
+            "j": ("j", [0]),
+        },
+    )
+
+
+@pytest.fixture
+def absorbing_only(tmpdir):
+    """Absorbing only particles radiative properties data set path fixture."""
+    mu = np.linspace(-1.0, 1.0) * ureg.dimensionless
+    w = np.linspace(279.0, 2401.0) * ureg.nm
+    phase = np.stack([np.ones_like(mu) for _ in w], axis=0) / ureg.steradian
+    # Add trailing dimensions for column and row indices:
+    phase.shape += (1,) * (4 - phase.ndim)
```

Comment:
That's a peculiar way to reshape this array, I didn't know that an array's shape could be modified like this. However the final shape of the array is difficult to understand, is `np.reshape()` really not an option? (Don't forget that -1 can be used to guess one dimension.)

Reply:
Indeed, I agree that using `np.reshape()` would make the code easier to read